### PR TITLE
document yaml

### LINKED ISSUE #1435: Create docs for YAML configuration
All our experiments are configured in [YAML files](https://github.com/mozilla/testpilot/tree/master/content-src/experiments) now. Docs giving a short description of each value could be useful.

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #1435

### CODE REVIEW

File: docs/experiments/experiment-content/getting_started.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## How to build a new experiment
+
+1. Copy the `template.yaml` file below into `./content-src/experients`
```
Comment (ckprice) [inline docs/experiments/experiment-content/getting_started.md]: `experients`

File: docs/experiments/experiment-content/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+2. Rename your file to match the eventual slug of your experiement eg `tab-center.yaml`
```
Comment (ckprice) [inline docs/experiments/experiment-content/getting_started.md]: `experiement`

**Review by ckprice — COMMENTED** [review]
grammrar

**Review by ckprice — APPROVED** [review]

File: docs/experiments/experiment-content/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+4. In the directory you've created, `mkdir details social avatars tour icon`. You'll put various image assets into these folders.
+5. As you add images, you'll need to manually compress them. You can use an [app](https://imageoptim.com/mac) or a [command line tool](https://www.npmjs.com/package/image-min).
+
+### What goes in the image directories
+
+* `icon` 192px x 192px png icon, all white & transparent - experiment icon
+* `avatars` 64px x 64px jpg images - creator avatars
+* `details` 1280px x 720-1080px jpg images - appear on the experiment details page
+* `tour` 1280px x 720px jpg images - the experiment tour
+* `social` 1200px × 630px jpg image for facebook & 560px × 300px image for twitter
```
Comment (pdehaan) [inline docs/experiments/experiment-content/getting_started.md]: I'd change these to Facebook and Twitter proper, because trademarks and whatnot.

Comment (johngruen) [reply docs/experiments/experiment-content/getting_started.md]: classic @pdehaan

**Comment by pdehaan** [comment]
LGTM. Once this lands, and we do https://github.com/mozilla/testpilot/issues/1308, we should make sure we lint this template as well to make sure everything is kosher and in sync.

**Pull request merged**